src/types/utils/EmptyObject.ts

```diff
@@ -0,0 +1,15 @@
+import Falsy from './Falsy';
```

Comment:
Can you update this file to match [here](https://github.com/Expensify/App/blob/a43fd7f04c4e770ad278537e900b41265ce04331/src/types/utils/EmptyObject.ts)?

Reply:
I've used same function. naming as [here](https://github.com/Expensify/App/blob/abe189c11754f8731fc67632e1a9a33a682422d1/src/types/utils/EmptyObject.ts), since it is less confusing according to its way of using